engine/admin/logging/overview.md

```diff
@@ -243,13 +243,13 @@ logging driver options.
 
 For example, to specify both additional options:
 
-```bash
+```bash{% raw %}
```

Comment:
I had better success with this on a new line under the gate line, and with the endraw on a new line before the end gate line.